domains/justincodingprojects.is-cool.dev.json

```diff
@@ -12,18 +12,8 @@
     },
 
    "record": {
-    "A": [
-      "185.199.108.153",
-      "185.199.109.153",
-      "185.199.110.153",
-      "185.199.111.153"
-    ],
-
-    "AAAA": [
-      "2606:50c0:8000::153",
-      "2606:50c0:8001::153",
-      "2606:50c0:8002::153",
-      "2606:50c0:8003::153"
+    "CNAME": [
```

Comment:
CNAME should be a string.

Reply:
oh, my bad

Reply:
i forgot to remove the brackets when renaming "A" to "CNAME"